src/lib/php-get-theme-details.ts

```diff
@@ -1,37 +1,42 @@
-import * as Sentry from '@sentry/electron/main';
+import { z } from 'zod';
 import SiteServerProcess from 'src/lib/site-server-process';
 
+export const themeDetailsSchema = z.object( {
```

Comment:
I don't think we need to export this schema.

```suggestion
const themeDetailsSchema = z.object( {
```

Reply:
Thanks! That was a leftover from me playing around with where to put this schema